tests/transform/test_dynamodb_types.py

```diff
@@ -11,6 +11,7 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 import unittest
+from collections import Counter
```

Comment:
I never used `collections.Counter` before, so I will probably need to ramp up on that. Thanks!

Reply:
That's really versatile and very useful. Thank you!

-- https://docs.python.org/3/library/collections.html#counter-objects